src/microbe_stage/MicrobeHUD.cs

```diff
@@ -524,18 +526,27 @@ public void UpdateEnvironmentalBars(BiomeConditions biome)
 
         oxygenBar.MaxValue = 100;
         oxygenBar.Value = oxygenPercentage;
-        oxygenBar.GetNode<Label>("Value").Text = oxygenPercentage + "%";
+
+        // {0}%
```

Comment:
I'm leaning a bit towards that one number + `%` is something that doesn't need localization.
Do you know of any case where `5%` would need to be localized as something else than that?